src/TobiasOlry/Talkly/Controller/TopicController.php

```diff
@@ -3,7 +3,6 @@
 namespace TobiasOlry\Talkly\Controller;
```

Comment:
Since #53 in TopicController line 45 + 49 will break while trying to call toArray() on a normal Array.